tests/unit/converter/ena/test_ena_experiment.py

```diff
@@ -0,0 +1,115 @@
+import unittest
+
+from converter.ena.ena import HcaData
+from converter.ena.ena_experiment import EnaExperiment, HcaEnaMapping
+from xsdata.formats.dataclass.serializers import XmlSerializer
+from xsdata.formats.dataclass.serializers.config import SerializerConfig
+
+
+class TestEnaExperiment(unittest.TestCase):
+    config = SerializerConfig(pretty_print=True)
```

Comment:
Could an `EXPERIMENT.XML` contain more than 1 experiment?
If yes, then it would be nice to cover that case.
Tx